content/references/markdown/index.md

```diff
@@ -4,12 +4,164 @@ topic: markdown
 next: /references/markdown/extensions/
 ---
 
+
+
 # Basic Markdown Syntax
 
 Codewars supports [Markdown][wiki-markdown].
 
 More specifically, [GitHub Flavored Markdown][gfm] (strict superset of [CommonMark][common-mark]) and few [Codewars extensions][extensions].
 
+You may find below a summary of the most used ones.
+
+
+
+
+## Emphasis
+
+```markdown
+_This is italic_
+*This is also italic*
+
+__This is bold__
+**This is bold too**
+
+___here, italic and bold___
+***here, italic and bold***
+```
+Note that `*You **can** combine them*`: *You **can** combine them*.
+
+
+
+
+## Headers
+
+```markdown
+# This is &lt;h1&gt;
+## This is &lt;h2&gt;
+###### This is &lt;h6&gt;
+```
+
+
+
+## Code blocks
+
+To get text displayed without the renderer playing with the indentation or whitespaces, code blocks
+
+### Inline code blocks:
+
+To highlight code inside a sentence, use simple backticks  ``` `print("This is some code")` ```.
+
+### Mutliline code blocks:
```

Comment:
~~~suggestion
## Displaying code

Use code spans and code blocks to show the text as is. Use code block to show the syntax highlighted code.

### Code span

Use single backtick (<code>`</code>) to show the text as is inline.

```markdown
Code span example: `print("example")`
```

### Code block
~~~